plugins/subnet/help.md

```diff
@@ -1,130 +1,133 @@
 # Description
 
-The Subnet plugin takes input as a network in CIDR notation and returns useful information, such as the number of hosts, the ID, host address range, and broadcast address.
+The Subnet plugin takes input as a network in CIDR notation and returns useful information, such as the number of hosts, the ID, host address range, and broadcast address
 
 # Key Features
 
 * Returns network information about CIDR
 
 # Requirements
-
-_This plugin does not contain any requirements._
+  
+*This plugin does not contain any requirements.*
 
 # Supported Product Versions
 
-_There are no supported product versions listed._
+* 2024-10-09
 
 # Documentation
 
 ## Setup
-
-_This plugin does not contain a connection._
+  
+*This plugin does not contain a connection.*
 
 ## Technical Details
 
 ### Actions
 
-#### Check Address in Subnet
 
-This action is used to determine if the provided IP address is in the subnet.
+#### Calculate
 
-##### Input
+This action is used to return subnet information for IP and Netmask
```

Comment:
help validator is failing on this line for some reason

Reply:
It's because it contains `subnet` as the plugin name in the description. I would ignore it in that case 😄